docs/Deprecated-easyconfigs.rst

```diff
@@ -0,0 +1,53 @@
+.. _deprecated_easyconfigs:
+
+Deprecated easyconfigs
+======================
+
+Since EasyBuild v3.8.0, individual easyconfig files or particular (versions of) toolchains can be marked as deprecated.
+
+.. contents::
+    :depth: 3
+    :backlinks: none
+
+
+.. _deprecated_easyconfigs_symptoms:
+
+Symptoms
+--------
+
+Using an easyconfig file or toolchain that was marked as deprecated results in a warning message like::
+
+  WARNING: Deprecated functionality, will no longer work in v4.0: easyconfig file '/home/example/test.eb' is marked as deprecated:
```

Comment:
These blocks are being syntax highlighted (for Python I think). Not a sphinx expert but should be easy to switch off.

Reply:
You would expect this to be easy to fix, turns out it's not...

With `.. code:: none`, the block is not displayed at all, with `.. code:: text` I'm still getting highlighting.

I'll keep poking at it, thanks for pointing this out, it indeed looks a bit silly this way...